src/main.rs

```diff
@@ -1,6 +1,10 @@
 //! # wiki
 #![deny(missing_docs)]
 
+#[cfg(test)]
```

Comment:
Cargo provides a functionality to put tests into the `./tests/` directory that we can separate the tests from our main application. I would recommend to do that.